pkg/sriov/network.go

```diff
@@ -353,6 +360,23 @@ func (builder *NetworkBuilder) withCapabilities(capability string) *NetworkBuild
 	return builder
 }
 
+func (builder *NetworkBuilder) withIpam(ipamType string) *NetworkBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
```

Comment:
Please check if ipamType is not empty string

Reply:
Added verification